tests/e2e/lib/setup-env.js

```diff
@@ -80,19 +80,39 @@ export const catchBeforeAll = async ( callback, errorHandler = defaultErrorHandl
 };
 
 let video;
-let captureVideo = process.env.CAPTURE_VIDEO === 'true';
+const captureVideo = process.env.CAPTURE_VIDEO === 'true';
 
 async function setupBrowser() {
-	await page.setViewportSize( { width: 1280, height: 1024 } );
+	let userAgent = await page.evaluate( () => navigator.userAgent );
+	const userAgentSuffix = 'wp-e2e-tests';
+
+	// Only update user agent if it wasn't already previously updated
```

Comment:
Since userAgent is constant, and it won't change in runtime, I think it is safe to use `jestPlaywright`s config file to set up a userAgent. `contextOptions`  should be capable to handle custom `userAgent`

Reply:
Actually it's not constant in our case. I tried to replicate the exact same concept as used in the Puppeteer implementation, where the user agent was changed at runtime by appending 'wp-e2e-tests' to the browser's user agent string.
If we don't care to use a real user agent of the real browser instance we're using in the test, then yes, hardcoding it in a config makes sense.